packages/terra-functional-testing/src/commands/viewport-helpers.js

```diff
@@ -0,0 +1,73 @@
+const Logger = require('../logger/logger');
+const terraViewports = require('../util/viewports');
+
+const logger = new Logger({ prefix: 'wdio-terra-service' });
+
+/**
+* Convenience method for getting the Terra test viewports by name.
+*
+* @param {string|string[]} sizes - the list of Terra viewport names.
+* @return {object} - Terra viewport sizes.
+*/
+const getViewports = (...sizes) => {
+  let viewportSizes = Object.keys(terraViewports);
+  if (sizes.length) {
+    viewportSizes = sizes;
+  }
+  return viewportSizes.map(formFactor => terraViewports[formFactor]);
```

Comment:
Are `formFactor` and `viewport` synonyms or is `formFactor` always the 'tiny, medium, small' and viewport the value (width, height)? The words seem interchangeable throughout this file.

Reply:
`formFactor` is referring more to the FORM_FACTOR set in the env variable whereas `viewport` is referring to the window sizes we define. But yes, these seem to be used interchangeably. I've updated the variable names here and comment too `viewpoort` whenever referring to what we defined and use `formFactor` when referring to the env option. https://github.com/cerner/terra-toolkit/pull/438/commits/402ee523e9eabedfc323de69d40318f8b075dd7d